api/blog.go

```diff
@@ -22,6 +22,7 @@ func ListPosts(client string, page string) (Posts, int64, error) {
 	var buf bytes.Buffer
 
 	pageInt, err := strconv.Atoi(page)
+	sizeInt, err := strconv.Atoi(size)
```

Comment:
Check err != nil for size int too 

if err!= nil {
log.Fatalf("Error converting page", err)
}
sizeInt, err := strconv.Atoi(size) 
if err!= nil {
log.Fatalf("Error converting size", err)
}